data/en/test_routing_not_affected_by_answers_not_on_path.json

```diff
@@ -0,0 +1,140 @@
+                "title": "",
+                "description": "Enter an answer and continue",
+                "questions": [{
+                    "answers": [{
+                        "id": "invalid-path-answer",
+                        "mandatory": true,
+                        "type": "Number"
+                    }],
+                    "id": "invalid-path-question",
+                    "title": "Enter a number",
+                    "type": "General"
+                }]
+            }, {
+                "type": "Interstitial",
+                "id": "invalid-path-interstitial",
+                "title": "You now have an answer that could be invalid in the store.",
+                "description": "Go back to the first question and choose the second path.",
+                "routing_rules": [{
+                    "goto": {
+                        "block": "summary"
+                    }
+                }]
+            }, {
+                "type": "Question",
+                "id": "valid-path",
+                "title": "Route page",
+                "description": "This page should take you to the valid page. If it takes you to the invalid page then routing is using an answer that isn't on the routing path",
+                "questions": [{
+                    "answers": [{
+                        "id": "valid-path-answer",
+                        "mandatory": true,
+                        "type": "Number"
+                    }],
+                    "id": "valid-path-question",
+                    "title": "Enter a number and continue",
+                    "type": "General"
+                }],
+                "routing_rules": [{
+                    "goto": {
+                        "block": "invalid-final-interstitial",
+                        "when": [{
+                            "id": "invalid-path-answer",
+                            "condition": "set"
+                        }]
+                    }
+                }, {
+                    "goto": {
+                        "block": "valid-skipped-interstitial"
+                    }
+                }]
+            }, {
+                "type": "Interstitial",
+                "id": "valid-skipped-interstitial",
+                "title": "This page should have been skipped!",
+                "description": "",
+                "skip_conditions": [{
+                    "when": [{
+                        "id": "invalid-path-answer",
+                        "condition": "not set"
+                    }]
+                }]
+            }, {
+                "type": "Interstitial",
+                "id": "valid-final-interstitial",
+                "title": "You were routed correctly!",
+                "description": ""
+            }, {
+                "type": "Interstitial",
+                "id": "invalid-final-interstitial",
+                "title": "You were routed incorrectly.",
+                "description": ""
```

Comment:
This doesn't work, it's always shown even if the `valid-final-interstitial` page is shown.